var/spack/repos/builtin/packages/xfsdump/package.py

```diff
@@ -25,8 +25,10 @@ class Xfsdump(MakefilePackage):
     depends_on("attr")
     depends_on("xfsprogs@:4.20.0")
 
-    def setup_build_environment(self, env):
-        env.append_flags("LDFLAGS", "-lintl")
+    def flag_handler(self, name, flags):
+        if name == "ldlibs" and "intl" in self.spec["gettext"].libs.names:
+            flags.append("-lintl")
+        return build_system_flags(name, flags)
```

Comment:
Note that `MakefilePackage` does not implement `flags_to_build_system_args` (like `CMakePackage` does), so this may not have any effect: I think you may want to use `env_flags` here.

Reply:
Yep, that edit was probably a mixup with `xfsprogs`: will fix.